bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -58,15 +58,36 @@ def __init__(self, context, request, **kwargs):
             }
         self.review_states = [
             {'id': 'default',
-             'title':  _('Results pending'),
+             'title': _('Assignment pending'),
+             'transitions': [{'id': 'submit'},
+                             {'id': 'cancel'},
+                             # {'id': 'assign'}
+                             ],
+             'contentFilter': {
+                 'review_state': ['sample_received'],
+                 'cancellation_state': 'active', },
+             'columns': ['AnalysisRequest',
+                         'Worksheet',
+                         'Service',
+                         'Result',
+                         'Uncertainty',
+                         'Partition',
+                         'Method',
+                         'Instrument',
+                         'Analyst',
+                         'state_title',
+                         ]
+             },
+            {'id': 'results_pending',
+             'title': _('Results pending'),
              'transitions': [{'id': 'sample'},
```

Comment:
Remove `sample` and `cancel` transitions. At this state (Results pending), only `submit` transition should be allowed.